README.md

```diff
@@ -20,7 +20,6 @@ This project also provides the documented Operator Collection specification, alo
 - [Operator Collection SDK](ibm/operator_collection_sdk/README.md)
 - [Specification](docs/spec.md)
 - [Tutorial](docs/tutorial.md)
-- [Example Operator](examples/racf-operator/)
```

Comment:
Why do you want to remove this link to the RACF-Operator example? I thought it was a good link to have, as I started from that link.

Reply:
@zohiba I received feedback that having a tutorial and example operator link was confusing and causing users to initially navigate to the example operator instead of the tutorial. And once they were viewing the example, they were unsure what to do next.....which the goal is for users to walk through the tutorial to execute the steps to deploy this operator.